spec/policies/application_policy_spec.rb

```diff
@@ -2,10 +2,30 @@
   subject { described_class }
 
   permissions :update?, :index?, :show?, :create?, :new?, :edit?, :destroy? do
+    context 'when is an user' do
+      let!(:user) { create(:user) }
```

Comment:
We can just `build` here
```suggestion
      let(:user) { build(:user) }
```